pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -91,18 +91,43 @@ var (
 		"Indicates the IP reuse duration in millisecond for all IPs.",
 		nil, nil,
 	)
+	assignedIPv4AddrCountDesc = prometheus.NewDesc(
+		"ipv4_assigned_count",
+		"Indicates the total Ipv4 IPs assigned to the subnetwork.",
+		nil, nil,
+	)
+	assignedIPv6AddrCountDesc = prometheus.NewDesc(
+		"ipv6_assigned_count",
+		"Indicates the total Ipv6 IPs assigned to the subnetwork.",
+		nil, nil,
+	)
+	IPv4AddrUtiliztaionDesc = prometheus.NewDesc(
```

Comment:
How did we decide to export the utilization ratio instead of utilization count? By the way, typo here.